app/src/main/java/me/nikhilchaudhari/compose_particle_system/MainActivity.kt

```diff
@@ -9,6 +9,8 @@ import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.runtime.Composable
```

Comment:
Could you please also share a video/gif using particle image bitmap 😄? I could add this to README.md then.

Reply:
https://gfycat.com/thankfulmeaslygroundbeetle
There is a short video here of sleeping cat particles being fired up using the fountain method 😸

Reply:
Awesome!